tern/classes/image_layer.py

```diff
@@ -295,6 +296,15 @@ def gen_fs_hash(self):
             rootfs.extract_tarfile(tar_file, fs_dir)
             self.__fs_hash = rootfs.calc_fs_hash(fs_dir)
 
+    def get_layer_hash(self):
+        '''Get the layer hash from the name of the directory that holds
+        layer.tar. This comes from the layer.tar_file property if the image
+        class was created with a tar_file.'''
+        if self.__tar_file:
+            # extract from <fs path>/tarball_hash/layer.tar
+            return self.__tar_file[:-64]
+        return ''
+
```

Comment:
I may have thought this was more straightforward than it looks. "layer.tar" is a Docker thing. OCI images have a different way of storing the layers i.e. their digest is in the config "layers" and they correspond to the hash of the tarball or whatever artifact. Could you move this function into the DockerImage class, and set the layer checksum property with the result of this and then replace `layer.fs_hash[:10]` with `layer.checksum`?

Reply:
Had some discussion with Nisha about this. Instead of trying to pull the layer hash from the variety of container image client manifests (that all represent the hash in different ways/places) we will refer to the layers by order. We feel that this is more widely understood and intuitive.